src/llm/llm.service.ts

```diff
@@ -42,12 +43,28 @@ export class LlmService {
     private readonly conversationStateModel: typeof ConversationState
   ) {}
 
-  private enhanceMessagesWithToolContext(
+  private async enhanceMessagesWithToolContext(
     previousMessages: LLMContext[],
-    lastToolCalls: ConversationState['last_tool_calls']
-  ): LLMContext[] {
+    lastToolCalls: ConversationState['last_tool_calls'],
+    channelId: string,
+    threadTs?: string,
+    slackWorkspaceDomain?: string
+  ): Promise<LLMContext[]> {
     const enhancedPreviousMessages = [...previousMessages];
 
+    if (slackWorkspaceDomain) {
+      const slackUrl = getSlackMessageUrl({
+        slackDomain: slackWorkspaceDomain || '',
```

Comment:
Falsy check is already present above
```suggestion
        slackDomain: slackWorkspaceDomain,
```

Reply:
Yes, the additional check is not required again since we are already checking it in the if condition. I'll update it.